irrd/daemon/main.py

```diff
@@ -142,10 +142,8 @@ def run_irrd(mirror_frequency: int, config_file_path: str, uid: Optional[int], g
 
     mirror_scheduler = MirrorScheduler()
 
-    preload_manager = None
-    if not get_setting(f"database_readonly"):
```

Comment:
Should we run the preloader in non-standby database_readonly mode? What is actually the purpose of database_readonly without standby? Perhaps we should not have separate settings.